digiroad2-oracle/src/main/resources/db/migration/V167__add_table_to_feedback.sql

```diff
@@ -0,0 +1,12 @@
+create table feedback
+(
+  id number primary key,
+  receiver varchar2(128),
+  createdBy varchar2(128),
+  createdAt timestamp default current_timestamp not null,
+  subject varchar2(128),
+  body varchar2(4000),
+  status char(1) default '0',
+  statusDate timestamp default current_timestamp,
+  CONSTRAINT status check (status in ('1','0'))
+);
```

Comment:
could be replace by

create table feedback
(
  id number primary key,
  receiver varchar2(128),
  created_By varchar2(128),
  created_Date timestamp default current_timestamp not null,
  subject varchar2(128),
  body varchar2(4000),
  status char(1) default '0',
  status_Date timestamp default current_timestamp,
  CONSTRAINT status check (status in ('1','0'))
);